conftest.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright (c) 2017, Anaconda, Inc. All rights reserved.
+#
+# Licensed under the terms of the BSD 3-Clause License.
+# (See LICENSE.txt for details)
+# -----------------------------------------------------------------------------
+import functools
+
+import pytest
+
+
+@pytest.fixture(params=["packages", "dependencies"])
+def pkg_key(request):
+    """Ensure equivalence between `dependencies` and `packages`"""
+    yield request.param
+
+
+def _change_default_pkg_key(test_function):
```

Comment:
No, it should probably be used for some others too that also use the `pkg_key` fixture to ensure that explicitly using both the default value, and the alternative value works in both cases of default value -- 4 cases.  In practice, it is hard for me to imagine a situation where a test works with two of the cases, but fails with the other two, but I can't rule this out, so I provide this in case we stumble on one.